listenbrainz/replay_user.py

```diff
@@ -0,0 +1,97 @@
+""" This script should be used to replay user listens to fix bad data.
+"""
+import json
+import uuid
+
+from datetime import datetime
+from flask import current_app
+from listenbrainz.utils import get_measurement_name, quote, convert_to_unix_timestamp
+from listenbrainz.webserver import create_app
+from listenbrainz.webserver.influx_connection import init_influx_connection
+from listenbrainz.listenstore.influx_listenstore import DUMP_CHUNK_SIZE
+
+
+class UserReplayer:
+    def __init__(self, user_name):
+        self.user_name = user_name
+        self.max_time = datetime.now()
+        self.app = create_app()
+
+
+    def filter_function(self, row):
```

Comment:
I think we should make this an abstract function that must be overriden. Right now this could effectively be the same as a quiet failure...

Reply:
In fact, it might be nice to provide a skeleton example script that overrides the abstract function and serves as a template for replaying other users.